docs/conf.py

```diff
@@ -40,7 +48,24 @@
 # -- Options for HTML output ----------------------------------------------
 
 # We want to set the RTD theme, but not if we're on RTD.
-if os.environ.get('READTHEDOCS', '') != 'True':
+if os.environ.get('READTHEDOCS', '') == 'True':
```

Comment:
Craig did it, I just changed the existing stuff in `docs/conf.py`.

I'll do whatever you like, just let me know.